shared/actions/chat/search.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import * as Creators from './creators'
+import * as Constants from '../../constants/chat'
+import * as Selectors from '../../constants/selectors'
+import * as SearchConstants from '../../constants/search'
+import * as SearchCreators from '../search/creators'
+import * as Saga from '../../util/saga'
+import type {ReturnValue} from '../../constants/types/more'
+import type {TypedState} from '../../constants/reducer'
+
+const inSearchSelector = (state: TypedState) => state.chat.get('inSearch')
+
+function* _newChat(action: Constants.NewChat): Saga.SagaGenerator<any, any> {
```

Comment:
:sheep::sheep: should this maybe go in `manage-thread.js`?

Reply:
i think this stuff will move again once this is all done. the first pass at this is to logically leave this all the same and just move things around. next pass will be to simplify the logic

Reply:
cool, sounds good